projects/opentrain/gtfs_pipeline/common/agency.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";

```

Comment:
It would be better if you merge all the proto files.
You can have several messages in a single proto file.